include/types/HttpReply.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef HTTPREPLY_HPP
+#define HTTPREPLY_HPP
+
+#include <string>
+#include <unordered_map>
+
+namespace api
+{
+    /**
+     * @struct HTTPReply
+     * @brief This structure defines an example type for HTTP(S) responses.
+     */
+    struct HTTPReply
+    {
+        /**
+         * @brief This field refers to the response code.
+         */
+        std::uint16_t code;
+
+        /**
+         * @brief This string refers to the response body.
```

Comment:
@brief Refers to [...]

Reply:
same as the previous one I'd go with only "The response body"